code_survey/models/survey_user_input.py

```diff
@@ -34,7 +44,8 @@ def _fetch_tests(self, question, is_test=False):
             if is_test and test.is_hidden:
                 continue
             this_test_locals = self._get_locals_dict(test.input_)
```

Comment:
Same here, test it out to be sure, but you might be able to use:
```suggestion
            this_test_locals = test._get_locals_dict(test.input_)
```
And maybe even inside the _get_locals_dict you could use self to refer to the test object :+1:

Reply:
Ohh makes sense